src/__tests__/__snapshots__/LabelGroup.js.snap

```diff
@@ -0,0 +1,81 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`BranchName matches snapshot 1`] = `
+.c0 .LabelItem:last-child {
+  margin-right: 0;
+}
+
+.c1 {
+  display: inline-block;
+  font-weight: 600;
+  line-height: 1;
+  color: #fff;
+  border-radius: 2px;
+  font-size: 12px;
+  padding-left: 3px;
+  padding-right: 3px;
+  padding-top: 4px;
+  padding-bottom: 4px;
+  margin: 4px;
+  background-color: #6a737d;
+  width: medium;
+  height: medium;
+}
+
+.c1:hover {
+  -webkit-text-decoration: none;
+  text-decoration: none;
+}
+
+.c2 {
+  display: inline-block;
+  font-weight: 600;
+  line-height: 1;
+  color: #fff;
+  border-radius: 2px;
+  font-size: 12px;
+  padding-left: 3px;
+  padding-right: 3px;
+  padding-top: 4px;
+  padding-bottom: 4px;
+  margin: 4px;
+  background-color: #6a737d;
+  width: medium;
```

Comment:
This `size: medium` style is due to a sort of bug @BinaryMuse and I discovered which she is currently working on a patch for. It's been happening for any component who uses a prop called `size` and doesn't have any side-effects so I'm fine merging this before Michelle's fix is out (though her fix will probably end up in the `release-14.4.0` branch as well